payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -40,6 +44,14 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         updateBalance(updatedToWalletBalance, toWallet);
 
         executorService.execute(() -> notificationClient.sendNotification(walletPayment));
+        transactionHistoryClient.saveTransactionToHistory(new TransactionCreation(
```

Comment:
create data using builder in a dedicated variable.
Also move this data and request itself to a separate method